### ISSUE TITLE
add support for the "target_commitish" property

### ISSUE THREAD

**Issue by davelandry** [coderaiser/node-grizzly#5]
Helpful when creating a release on a non-master branch. Here's the docs: https://developer.github.com/v3/repos/releases/#create-a-release

**Comment by coderaiser** [comment]
What a great idea for a `pull request` :).
Command line arguments could be like this:

``` sh
 -tc, --target-commitish        value that determines where the Git tag is created from
```

**Comment by davelandry** [comment]
Duh, my bad 😏 

#6

**Comment by coderaiser** [comment]
Landed in [v2.1.0](https://github.com/coderaiser/node-grizzly/releases/tag/v2.1.0).
